### PR TITLE
Add xDai category + articles

### PR DESCRIPTION
**Pull request by wtzb**

### CODE REVIEW

File: general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+2. Select the wallet you’ll be connecting to MyCrypto. _Note: click [View Address](/how-to/accessing-wallet/how-does-view-address-work) if you only plan to track an account and not send from it._
+3. Select the xDai network from the dropdown. _Note: If you’re using MetaMask, you will not see this dropdown and you’ll need to make sure you’ve selected the xDai network within MetaMask before continuing._
+
+![Select network](../../assets/general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto/select-network.png)
+
+4. Continue to connect, and check out your xDai account(s) on your MyCrypto dashboard, right alongside any other accounts you have!
+
+![Accounts overview](../../assets/general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto/accounts-overview.png)
+
+As mentioned earlier, you can use the same account for Ethereum and for xDai, so we’ve made sure to distinguish your accounts separately according to the network they’re attached to. Note the above screenshot where it shows two separate listings for one address — one used with Ethereum, one used with xDai.
```
Comment (jspence425) [inline general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md:21]: "As mentioned earlier" > "As mentioned [in this article](url)" and please link to the appropriate article where it's mentioned. I think it's the 'what is xdai' article

Comment (jspence425) [reply general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md:21]: nope jk its referenced in the the 'how to get xdai article'

**Review by jspence425 — CHANGES_REQUESTED** [review]
Left a comment. The rest looks good to me!

File: general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md (hunk 1 above)
Comment (wtzb) [reply general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md]: Thanks!

**Review by Mrtenz — APPROVED** [review]

File: general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ If you want one place to view and manage all of your xDai accounts at once, you
 
 ![Accounts overview](../../assets/general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto/accounts-overview.png)
```
Comment (Butterflies22) [inline general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md:19]: how do i do this crap

**Review by Butterflies22 — COMMENTED** [review]
jolean

**Pull request merged**